src/project-css/app/Http/Controllers/UserController.php

```diff
@@ -70,8 +66,7 @@ public function promote(Request $request){
     }
 
     // Error message
-    $thisErrMsg = "The user names doesn't match";
-    return redirect()->route('userIndex')->withErrors($thisErrMsg);
+    return redirect()->route('userIndex')->withErrors("The user names doesn't match");
```

Comment:
Means nothing to the user, what does it actually mean.   Ie:: Your username isn't correct?  We have you as a separate user?  You don't have permission to add a user?  We need something more pertinant.

Reply:
updated error messages